examples/isolated-variables/isolated_variables.bal

```diff
@@ -0,0 +1,72 @@
+import ballerina/io;
+
+type Coordinates record {|
+    decimal latitude;
+    decimal longitude;
+|};
+
+// The initialization expression of an `isolated` variable
+// has to be an `isolated` expression, which itself will be
+// an isolated root.
+// e.g., expressions with an immutable static type,
+// simple literals, constructor expressions
```

Comment:
```suggestion
// simple literals, and constructor expressions
```

Reply:
Same as previous comment, we're using "etc." at the end.